firefox_desktop/views/urlbar_events.view.lkml

```diff
@@ -0,0 +1,503 @@
+  type: string
+}
+
+dimension: normalized_country_code {
+  sql: ${TABLE}.normalized_country_code ;;
+  type: string
+}
+
+dimension: num_add_on_impressions {
+  hidden:  yes
+}
+
+dimension: num_admarketplace_sponsored_impressions {
+  hidden:  yes
+}
+
+dimension: num_bookmark_impressions {
+  hidden:  yes
+}
+
+dimension: num_chars_typed {
+  hidden:  yes
+}
+
+measure: average_characters_typed {
+    type: number
+    sql: AVG(${TABLE}.num_chars_typed) ;;
+  }
+
+dimension: num_default_partner_search_suggestion_impressions {
+  hidden:  yes
+}
+
+dimension: num_history_impressions {
+  hidden:  yes
+}
+
+dimension: num_navigational_impressions {
+  hidden:  yes
+}
+
+dimension: num_open_tab_impressions {
+  hidden:  yes
+}
+
+dimension: num_pocket_collection_impressions {
+  hidden:  yes
+}
+
+dimension: num_quick_action_impressions {
+  hidden:  yes
+}
+
+dimension: num_search_engine_impressions {
+  hidden:  yes
+}
+
+dimension: num_total_results {
+  hidden:  yes
+}
+
+dimension: num_trending_suggestion_impressions {
+  hidden:  yes
+}
+
+dimension: num_weather_impressions {
+  hidden:  yes
+}
+
+dimension: num_wikipedia_dynamic_impressions {
+  hidden:  yes
+}
+
+dimension: num_wikipedia_enhanced_impressions {
+  hidden:  yes
+}
+
+dimension: product_engaged_result_type {
+  hidden:  yes
+}
+
+dimension: product_first_result_type {
+  hidden:  yes
+}
+
+dimension: sample_id {
+  hidden:  yes
+}
+
+dimension: session_action_type {
+  hidden:  yes
+}
+
+dimension_group: submission {
+  sql: ${TABLE}.submission_date ;;
+  type: time
+  timeframes: [
+    raw,
+    date,
+    week,
+    month,
+    quarter,
+    year,
+  ]
+  convert_tz: no
+  datatype: date
+}
+
+  measure: event_count {
+    hidden: yes
+  }
+
+measure: urlbar_clicks {
+  group_label: "Urlbar Metrics"
+  sql: sum(case when ${is_terminal} and ${session_action_type} = "engaged" then 1 else 0 end);;
+  type: number
+}
+
+measure: urlbar_impressions {
+  group_label: "Urlbar Metrics"
+    sql: sum(case when ${is_terminal} then 1 else 0 end);;
+    type: number
+}
+
+measure: urlbar_CTR {
+  group_label: "Urlbar Metrics"
+  description: "Clicks / Impressions"
+    sql: safe_divide(${urlbar_clicks}, ${urlbar_impressions});;
+    type: number
+}
+
+measure: urlbar_annoyances {
+  group_label: "Urlbar Metrics"
+    sql: sum(case when ${session_action_type} = "annoyance" then 1 else 0 end);;
+    type: number
+}
+
+  measure: addon_clicks {
+    group_label: "Addon"
+    sql: sum(case when product_engaged_result_type = "add_on" and ${is_terminal} and ${session_action_type} = "engaged" then 1 else 0 end);;
+    type: number
+  }
+
+  measure: addon_impressions {
+    group_label: "Addon"
+    sql: sum(case when ${num_add_on_impressions} > 0 and ${is_terminal} then 1 else 0 end);;
+    type: number
+  }
+
+  measure: addon_CTR {
+    group_label: "Addon"
+    description: "Clicks / Impressions"
+    sql: safe_divide(${urlbar_clicks}, ${urlbar_impressions});;
```

Comment:
The variable names need to be updated here and in the CTR measure definitions below